password/src/Data/Password/Bcrypt.hs

```diff
@@ -183,6 +190,23 @@ checkPassword pass (PasswordHash passHash) =
       then PasswordCheckSuccess
       else PasswordCheckFail
 
+-- | Extract 'BcryptParams' from a 'PasswordHash' 'Bcrypt'.
+--
+-- Returns 'Just BcryptParams' on success.
+--
+-- >>> let pass = mkPassword "foobar"
+-- >>> passHash <- hashPassword pass
+-- >>> isJust $ extractParams passHash
```

Comment:
See comment at `Argon2` for the same advice.

`extractParams passHash == Just 10`